public/app/core/components/Signup/SignupPage.tsx

```diff
@@ -63,10 +64,11 @@ export const SignupPage: FC<Props> = (props) => {
           {({ errors, register, getValues }: FormAPI<SignupDTO>) => (
             <>
               <Field label="Your name">
-                <Input {...register('name')} placeholder="(optional)" />
+                <Input aria-label={selectors.pages.Signup.name} {...register('name')} placeholder="(optional)" />
```

Comment:
```suggestion
                <Input {...register('name')} id="user_name" placeholder="(optional)" />
```
Is there a reason for adding the aria labels here? I think adding `id` would make the inputs accessible enough, as then the label will be correctly associated with the inputs (our `Field` component adds `htmlFor` attribute [automatically](https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Forms/Field.tsx#L79-L88) for the input elements with `id`). We mostly use aria-labels in e2e tests and for components that do not have accessible role or name. This applies to all the other inputs in this component.

Reply:
sure, will do the cleanup and use `id` where ever i added `aria-label`

Reply:
addressed